Add new section for PhilipsTV binary_sensor (#27867)
home-assistant/home-assistant.github.io: source/_integrations/philips_js.markdown

  - Light
  - Media Player
  - Remote

>   - Binary Sensor

Reviewer: ```suggestion
  - Binary sensor
```
We're now doing sentence-style capitalization on the categories, too

  - media_player
  - remote
  - switch
  - sensor

>   - binary_sensor

Reviewer 2: PRs to the Core codebase may not contain multiple platforms in a single PR, therefore, this PR can also not contain multiple platforms being added in a single PR.

Author: Sure. Sorry about that. Understood the system now.

I changed both PRs to support binary_sensor only.
A new one will follow for sensor

Some TV's allow you to sync the processed ambilight color data to your Philips Hue bridge. This will make your Hue lights sync with the TV ambilight without the need to purchase a Hue Play HDMI Sync Box.
The integration exposes a "Ambilight+Hue" switch entity when your TV supports it which enables you to toggle this.

> ## Binary Sensor

Reviewer: ```suggestion
## Binary sensor
```

> Some newer OS version support live TV recording functions via the API.\
> For those TV'S this integration supports two entities:

Reviewer: ```suggestion
Some newer OS versions support live TV recording functions via the API.
For those TVs, this integration supports two entities:
```
